pontoon/base/views.py

```diff
@@ -295,6 +295,7 @@ def get_translations_from_other_locales(request):
 
     entity = get_object_or_404(Entity, pk=entity)
     locales = entity.resource.project.locales.exclude(code=locale)
+    selected_locales = request.user.profile.sorted_locales
```

Comment:
I doubt you'll be able to use `sorted_locales` and keep the order. Instead, you'll need to manually sort `preferred_locales` below.